pallets/protos/Cargo.toml

```diff
@@ -12,32 +12,32 @@ targets = ['x86_64-unknown-linux-gnu']
 
 [dependencies.sp-core]
 default-features = false
-git = 'https://github.com/paritytech/substrate.git'
-tag = 'monthly-2022-03'
+git = 'https://github.com/clamor-io/substrate.git'
```

Comment:
(https://github.com/clamor-io/substrate.git) is a fork of Substrate that is obviously owned by us, yes?

Reply:
Sort of, we are outsourcing that part but yeah we control.